utils/order.utils.js

```diff
@@ -0,0 +1,57 @@
+const { ORDER_ITEM_NOT_VALID } = require('../error-messages/orders.messages');
+
+function calculateTotalItemsPrice(items) {
```

Comment:
Move currency calculations to currency.utils from [PR#377](https://github.com/ita-social-projects/horondi_client_be/pull/377)